### PR TITLE
Setup supporting multiple architectures

### PR DESCRIPTION
**Pull request by nkrinner**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -442,9 +442,9 @@ function export_tftpboot_repos_dir()
     if iscloudver 5plus; then
         tftpboot_suse12_dir=/srv/tftpboot/suse-12.0
```
Comment (nkrinner) [inline scripts/qa_crowbarsetup.sh]: I think this stays unchanged, as we don't use multiple architectures on cloud 5?

Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: It needs to change as it's used to reference the directory that contains `install/`. So I guess we need a new code path specific to 6plus...

**Comment by vuntz** [comment]
I just realized one bit we will have to update too: https://github.com/nkrinner/automation/blob/multi-arch/scripts/qa_crowbarsetup.sh#L2351 (the code adding new repo to the provisioner barclamp). It has to contain the architecture as an attribute between the distro and the repos (suse-12.1 - x86_64 - SLES11)

**Comment by vuntz** [comment]
I sent https://github.com/nkrinner/automation/pull/2 with some potential improvements. Not tested yet, though.

**Pull request closed without merging**